tests/monitoring/monitoring_service/test_cli.py

```diff
@@ -36,6 +36,11 @@ def test_wrong_password(keystore_file, default_cli_args_ms):
     assert result.exit_code == 1
 
 
+def test_check_gas(web3):
+    private_key = '3a1076bf45ab87712ad64ccb3b10217737f7faacbf2872e88fdd9a537d8fe266'
+    check_gas_reserve(web3, private_key)
```

Comment:
this isn't checking anything. you should at least assert that the warning message is displayed.

Reply:
I see. I can't assert the log message since it is changing all the time according to the current gas estimate. Should we just remove that test or maybe test only that the log level is 'Warning'

Reply:
You can also only assert only on a part or create private key for an address with a fixed amount of eth.

Reply:
Yes good idea, but the gas_estimate always changes independently from the current balance. This gas estimate is then shown in the message. 
I could assert the len of the log message though, I will try

Reply:
Good point. For the log message see https://docs.pytest.org/en/latest/reference.html#caplog

Reply:
skipped the test, for now, fails with my setup at the moment.